### PR TITLE
Add production image build to this repository

### PR DESCRIPTION
**Pull request by sjmiller609**
Same idea as the Houston and Orbit PRs. Part of this epic: https://github.com/astronomer/issues/issues/465

- copied from astronomer/astronomer
- use COPY . . instead of git clone
- don't install gcloud because it's not used

Pre-merge checklist:
- [ ] Circle CI: Turn on for this repo
- [ ] Circle CI: Opt-in to third party orbs
- [ ] Circle CI: Add DOCKER_USERNAME DOCKER_PASSWORD to circle CI environment variables

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,49 +1,58 @@
 #
-# Commander
-# Astronomer Platform Provisioning Service
-
-FROM alpine:3.9
-MAINTAINER Astronomer <humans@astronomer.io>
+# Copyright 2018 Astronomer Inc.
```
Comment (andriisoldatenko) [inline Dockerfile]: 2019?

File: Dockerfile [hunk 2]
```diff
@@ -1,49 +1,58 @@
...
+	  && mkdir -p "${GOPATH}/bin" \
+	  && rm -rf /usr/lib/go \
+	  && apk del .build-deps
+
+ENTRYPOINT ["commander"]
+
+#
+# Build final image
+#
+FROM astronomerinc/ap-base:0.10.3
```
Comment (andriisoldatenko) [inline Dockerfile]: @sjmiller609 
how do you plan to upgrade ap-base version?

Comment (schnie) [reply Dockerfile]: We're breaking them apart. So we're going to only update this when an actual new version of `ap-base` is released. We're stopping the practice of tagging everything with the same release number all at once.

**Comment by sjmiller609** [comment]
@schnie this one is updated how we discussed

**Pull request merged**